activemodel/lib/active_model/secure_password.rb

```diff
@@ -61,23 +68,60 @@ def has_secure_password(options = {})
           raise
         end
 
+        attr_reader attribute
+
+        # Encrypts the password into the +password_digest+ attribute, only if the
+        # new password is not empty.
+        #
+        #   class User < ActiveRecord::Base
+        #     has_secure_password validations: false
+        #   end
+        #
+        #   user = User.new
+        #   user.password = nil
+        #   user.password_digest # => nil
+        #   user.password = 'mUc3m00RsqyRe'
+        #   user.password_digest # => "$2a$10$4LEA7r4YmNHtvlAvHhsYAeZmk/xeUVtMTYqwIvYY76EW5GUqDiP4."
```

Comment:
We're losing this documentation. Maybe we should push it up to the module on a fake `password=` attribute... or otherwise, we should merge it into the `has_secure_password` documentation. Either way, it's no good here.